Add version-specific BibTeX

albertvillanova opened the pull request:
As pointed out by @lhoestq in #2411, after the creation of the Zenodo DOI for Datasets, a new BibTeX entry is created with each release.

This PR adds a version-specific BibTeX entry, besides the existing one which is generic for the project.

See version-specific BibTeX entry here: https://zenodo.org/record/4817769/export/hx#.YLSyd6j7RPY

lhoestq commented:
Maybe we should only keep one citation ?
cc @thomwolf @yjernite

albertvillanova commented:
For info:
- The one automatically generated by Zenodo is version-specific, and a new one will be generated after each release.
- Zenodo has also generated a project-specific DOI, but currently this only redirects to the DOI page of the latest version.
- All the information automatically generated by Zenodo can be corrected/customized if necessary.

lhoestq commented:
Thanks for the details ! As zenodo says we should probably just show the versioned DOI. And we can remove the old citation.

albertvillanova commented:
I have removed the old citation.

What about the new one? Should we customize it? I have fixed some author names (replaced nickname with first and family names). Note that the list of authors is created automatically by Zenodo from this list: https://github.com/huggingface/datasets/graphs/contributors
I do not know if this default automatic list of authors is what we want to show in the citation...

lhoestq reviewed (approved):
Thank you :)
